domains/transfer-agreements/TransferAgreements.API/API.IntegrationTests/Controllers/TransferAgreementHistoryEntriesControllerTests.cs

```diff
@@ -0,0 +1,161 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Threading.Tasks;
+using API.ApiModels.Requests;
+using API.ApiModels.Responses;
+using API.Data;
+using API.IntegrationTests.Factories;
+using FluentAssertions;
+using VerifyTests;
+using VerifyXunit;
+using Xunit;
+
+namespace API.IntegrationTests.Controllers;
+
+[UsesVerify]
+public class TransferAgreementHistoryEntriesControllerTests : IClassFixture<TransferAgreementsApiWebApplicationFactory>
```

Comment:
There is a lot of scrubbed entries in the verified-files. Can the tests be written in a way, so it is only "AuditDate" that is being scrubbed?